wg-deployment/charter.md

```diff
@@ -0,0 +1,43 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Deployment is responsible for Kubeflow release and tooling to deploy apps from the catalog (e.g kfctl).
+
+### In scope
+
+#### Responsibilities
+
+- Provide tooling to deploy Kubeflow applications from the catalog.
+- Work with Working Group / Application Owners and Platform Owners to make the integration and release process easier.
+
+#### Code, Binaries and Services
+
+- kfctl (including operator)
+- manifests
```

Comment:
manifests should be removed since this will now be under the manifests WG.

Reply:
removed